deployments/cloudfoundry/tile/tile.yml

```diff
@@ -16,6 +16,10 @@ description: |
   This also includes a BOSH release of the Splunk OpenTelemetry Collector, called "splunk-otel-collector"
   that can be used in deployments to put the collector on individual VMs.
 
+stemcell_criteria:
+  os: 'ubuntu-jammy'
+  version: '1.148'
```

Comment:
What's this version reference?

Reply:
I see now, but I guess we're moving from https://network.tanzu.vmware.com/products/stemcells-ubuntu-xenial#/releases/1303059?

Reply:
Right, previously the default was `>= ubuntu-xenial 621.584`, but for some reason this rendered all stemcells invalid with the tile on TAS v3 and v4 (I tried xenial, bionic, and jammy based stemcells). I contacted the Pivotal team but have not heard back for the reasoning here.